docs/concepts/jcloud/yaml-spec.md

```diff
@@ -3,21 +3,19 @@
 
 JCloud extends Jina's {ref}`Flow YAML specification<flow-yaml-spec>` by introducing the special field `jcloud`. This lets you define resources and scaling policies for each Executor and gateway.
 
-Here's a Flow with two Executors that have specific resource needs. `indexer` demands 10G `ebs` disk, whereas `encoder` demands two cores, 8G RAM and two dedicated GPUs. 
+Here's a Flow with two Executors that have specific resource needs. `indexer` demands 10G `ebs` disk, whereas `encoder` demands G5 instance, which implies 4 cores and 8G RAM are used. See below sections for further information about Instance Type.
```

Comment:
```suggestion
Here's a Flow with two Executors that have specific resource needs: `indexer` requires a 10 GB `ebs` disk, whereas `encoder` requires a G5 instance, which implies that four cores and 8 GB RAM are used. See the below sections for further information about Instance Types.
```

Reply:
Let's use [Wikipedia abbreviation conventions](https://en.wikipedia.org/wiki/Gigabyte) for consistency